.gitignore

```diff
@@ -17,6 +17,7 @@
 
 # Untracked Directories
 .dev/
+.pseudo/
```

Comment:
Is there a  reason for adding this?

Reply:
Yes, I had started by stubbing out a pseudocode version which I put in here. I guess I could have put it in the `.dev` folder. I was going to add a comment but I figure once you know that it is a folder for actual pseudo-code, it's self-explanatory.